generation/classifier_free_guidance/2d_ddpm_classifier_free_guidance_tutorial.ipynb

```diff
@@ -0,0 +1,959 @@
bk+MLFiyQMYcPH06Od3R0yJi5c/U/Q6kqSe+YVLWjma4OHB4eljE5zai97amGz151olctmlMR51VJKl6FpKq29fYh557wrq2K8e4Vr2I157xWQhViDu83x/t+lvI3u9IaTvNkBwAIj2QHAAiPZAcACI9kBwAIj2QHAAiPZAcACG/aGkHnlLhWehlwzjHlxKimv8ejGjR7Jfetra3Jca903lv+oJYyeCXjah/M/KbTxW5vyZIlMmb+/PlyTjW3fuutt2RMT0+PnFPH5J0j1XTau06LFi2Sc6oZtXe+veUPVVVVck5RSwzKWdJeaeXzk1UJDesr7fzwZAcACI9kBwAIj2QHAAiPZAcACI9kBwAIb0qrMT2VVqlTiVQ1Wm7jWlV16VUG5jRa7urqknOq4nHx4sUyxquEXLt2bXL8Yx/7mIzZsmVLctyrTlSNkc10haJXneg1TX7hhReS4z/84Q9lzN/+9rfkeHt7u4xpbm6Wc8uWLUuO9/X1yZimpiY5p6o7VSWrmb6XvcrOnN+V3OrEnMbSmD482QEAwiPZAQDCI9kBAMIj2QEAwiPZAQDCI9kBAMKbU5hk3e3KlSunel/MbOY2Xp0K6lx4pddeufuRI0eS415jabUswdsHrwRdlf1/5StfkTFnn322nGtpaUmOeyXtOXKbb+dQS07mztX/bPrcc88lxx944AEZ8+STT8o5tbTEWyKimnybmTU2NibHvebW6jx497j3+1HqxtJqjt+wqeGd1/379x83nic7AEB4JDsAQHgkOwBAeCQ7AEB4JDsAQHgkOwBAeBW39ADH55X9q+7yZmb9/f3J8VWrVskYtfRAvUHBzOziiy+Wc3feeWdy3Lu/vLJ/tZzCK0/3SvgV72uSs73e3l45p5ZTeFRJ++joqIz5xS9+Iee2bduWHH/33XdljHeO1NsuvHM3MDBQdIw3p/Yv960HLD2oHG1tbcf9G57sAADhkewAAOGR7AAA4ZHsAADhkewAAOFRjVnBcirsvGqv+fPnJ8cPHjxYdMzpp58uY7Zv3y7nNm3alBzPqaLz5rzzoGK8fRgcHJRz6npUV1fLmIaGBjlX7OeY6SbHavx4uru7k+M33nijjHnhhRfknGoS7Z2H1tbW5HhfX5+M8a6huh655wiVg2pMAACMZAcAmAVIdgCA8Eh2AIDwSHYAgPBIdgCA8Ca99MBrFpzbSLVYld5gNacMPqdE3uOVUavmw17595YtW5LjDzzwgIzx7hWvibXildyr7Xnnob29PTn+l7/8Rcbs2rVLzi1dujQ5fujQIRmzefNmOTc+Pp4cP//882WMWiKSa2xsLDnuNeW+66675Nz3vve9orenlhgsX7686Bgzfa94+1Cu3zYcn3ct9u/ff9x4nuwAAOGR7AAA4ZHsAADhkewAAOGR7AAA4ZHsAADhzeq3HuQsZchZKpC7ZELNDQwMyJjm5mY5p8rxV69eLWN27tyZHP/ABz4gY7zO8wcOHEiOqw73ZmZ1dXVyrqenJzn+yCOPyJg//elPyXHvzQZLliyRc+vXr0+Oq07/Zma1tbVyLuc+UksZ1q5dK2PUfnufNTw8LGO86/6Nb3wjOf6d73xHxqjfHHXNzcwWLVok53LeDKGWYJj5x4vy4q0HAAAYyQ4AMAuQ7AAA4ZHsAADhkewAAOHpDqjTpJzNnnMqK3MqsLxjyjnelpYWGePtX2NjY3L8nnvukTGqmq+6ulrGqEbGZv6+K17T6eeeey45/u6778qY7u7u5LjXYFg1ezYz++tf/5ocv+SSS2SMV93Z39+fHPcqIT//+c8nx7/4xS/KmK997Wtybs2aNclxrzLWc8sttyTHVVNuM7Of//znyXGv4nhoaEjOqe+TV3FZU1Mj50pdfY2pxZMdACA8kh0AIDySHQAgPJIdACA8kh0AIDySHQAgvIpbelDppbmT7Jt9jNxjUp81OjoqYw4ePCjn7r333uT4RRddVNyOmV+u7R3v7t27k+O33367jPGOd+PGjcnxDRs2yJgXXnghOe6V9r/88styTi1zOPHEE2VMQ0ODnNu1a1dyXDXRNjPbsmVLcvz555+XMb29vXLuW9/6VnLca7RcVVUl5xYvXpwcv+OOO2TM73//++S4d++NjIzIObV/XiNoT85vAaYPT3YAgPBIdgCA8Eh2AIDwSHYAgPBIdgCA8OYUJllStGrVKjkXsSpJVRR6TY5VE+achtPenFc95lUh7ty5Mzl+0kknyRjF24d9+/bJua1btybHverEtrY2OdfU1JQc9xo3K3v37pVzXpXf+eefnxz3rq3XsFjth1eVqppYz58/X8Z49/JZZ52VHL/55ptljHeOvIbKynXXXZccf+qpp2RMfX29nFPnL/f7maPSq81nKu834j082QEAwiPZAQDCI9kBAMIj2QEAwiPZAQDCI9kBAMKruEbQla6cjaC90nDlyiuvlHMrVqxIjnvHpPZdLbMwM3vllVfk3Nq1a5PjR44ckTEtLS1ybuHChclxrxRZlaB3dHTImHPOOUfOtba2JsefeOIJGXPo0CE5t3r16uT4Bz/4QRmjGi339PTImJUrV8q55ubm5Pjg4KCM8cr+Fa/59he+8IXk+OOPPy5jGhsb5Vx1dXVy3Gss7WEZwczCkx0AIDySHQAgPJIdACA8kh0AIDySHQAgPJIdACC8SS89iPhmA4/q6l9VVSVjVDm+94YAjzrnXin+5s2b5Zza95xu9V7Z9fbt2+XcgQMHkuNeSbtXGq668B8+fFjGvPPOO8lx7w0B3jX8yU9+khz3rsXHP/5xObds2bLkuLcURR2vt7xg/fr1cq67uzs57i2Z8N6Mos6fdx9deumlyfFTTz1VxrS3txe9D9797y2xUd9PliRUJp7sAADhkewAAOGR7AAA4ZHsAADhkewAAOHN6kbQOVVTOTFeJZ9X3ZnThFlVO3pUxaWZbtDsHdOZZ54p5959993kuKpANDN77bXX5Jw6RyeccIKMUc2HvepEr9K
```

Comment:
To resolve the pep8 error: https://github.com/Project-MONAI/tutorials/actions/runs/11573173298/job/32214701041?pr=1867#step:7:304
```suggestion
    "    for _step, batch in progress_bar:\n",
```

Reply:
I just removed step. I was wrong in a previous comment, it's actually not needed